api-report/container-runtime.api.md

```diff
@@ -329,13 +327,15 @@ export interface ICompressionRuntimeOptions {
     readonly minimumBatchSizeInBytes: number;
 }
 
-// @public (undocumented)
+// @public @deprecated (undocumented)
 export interface IConnectableRuntime {
     // (undocumented)
     readonly clientId: string | undefined;
     // (undocumented)
     readonly connected: boolean;
     // (undocumented)
+    readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
```

Comment:
This was removed last week, and adding it back is the motivation for making this change, to avoid breaking this interface in a minor release -- even though we know now one should be using it.